core/domain/skill_services.py

```diff
@@ -489,6 +498,7 @@ def update_skill(committer_id, skill_id, change_list, commit_message):
     create_skill_summary(skill.id)
 
 
+
```

Comment:
Drop newline. There should be two blank lines between top-level function definitions.